plugins/woocommerce/client/admin/client/typings/global.d.ts

```diff
@@ -27,6 +30,11 @@ declare global {
 				currency?: {
 					symbol: string;
 				};
+				preloadSettings: {
```

Comment:
I guess it should be optional, since it used with `?`

Reply:
Yes, I think we can do that a18cece8fb97d614f4b2585c5722895237f7a99e